packages/user/src/views/Login.tsx

```diff
@@ -109,6 +117,18 @@ const Login: React.FC<IProperties> = ({
     );
   };
 
+  const showReminder = () => {
+    return (
+      <div className="reminder">
```

Comment:
return null in case `?redirect` param is empty or not present. We don't need to render empty `<div className="reminder">` all the time.